decksite/data/deck.py

```diff
@@ -488,15 +492,18 @@ def load_cards(decks: List[Deck]) -> None:
         return
     decks_by_id = {d.id: d for d in decks}
     sql = """
-        SELECT deck_id, card, n, sideboard FROM deck_card WHERE deck_id IN ({deck_ids})
+        SELECT deck_id, card, n_main, n_side FROM deck_card WHERE deck_id IN ({deck_ids})
```

Comment:
I think you can avoid all the database schema changing with a query like this:

```
SELECT
    SUM(CASE WHEN sideboard THEN 0 ELSE 1 END) AS n_main,
    SUM(CASE WHEN sideboard THEN 1 ELSE 0 END) AS n_side
FROM
    deck_card
WHERE
    deck_id IN ({deck_ids})
```

Does that do everything you need? Are there performance ramifications?

Reply:
The reason I wanted to do this change is because I couldn't figure out how to get `decksite/data/card.py:preaggregate_card()` to work. I think I was having trouble with the GROUPing. But I wil see if I can get it to work.

Reply:
I got it to work now, I don't recall exactly what the thing blocking me before was, but this works fine.